### PR TITLE
doc changes and release notes for release v2020.2.1

### PR DESCRIPTION
**Pull request by rotanid**
see also #2106

### CODE REVIEW

**Comment by rubo77** [comment]
> backports of batman-adv bugfixes

It would be interesting, what effect will those fixes have on gluon

**Comment by rotanid** [comment]
> > backports of batman-adv bugfixes
> 
> It would be interesting, what effect will those fixes have on gluon

as we don't post about Linux Kernel Fixes i like to say we also can't comment on minor batman-adv bugfixes. please read their release notes if you're interested.

**Comment by mweinelt** [comment]
> > backports of batman-adv bugfixes
> 
> It would be interesting, what effect will those fixes have on gluon

```
commit 4c05fe97d91a5bbce8f789526f05d512664796e6
Author: Sven Eckelmann <sven@narfation.org>
Date:   Mon Jul 6 20:04:33 2020 +0200

    batman-adv: Merge bugfixes from 2020.2
    
    * Revert "disable ethtool link speed detection when auto negotiation off"
    
    Signed-off-by: Sven Eckelmann <sven@narfation.org>
```

```
commit 822b55ff7d90e606c27051a782dd115b8cdc2505
Author: Sven Eckelmann <sven@narfation.org>
Date:   Mon Jul 6 20:04:33 2020 +0200

    batctl: Merge bugfixes from 2020.2
    
    * fix endianness when reading radiotap header
    * Only remove batadv interface on hardif reduction
    
    Signed-off-by: Sven Eckelmann <sven@narfation.org>
```

**Comment by rubo77** [comment]
@mweinelt, thanks for the hints.

But for me it is still difficult to understand the consequences on our network.

Would it be easy to explain the positive effect to a non-batman-developer in the release-notes? there are three possibilities:

- If the new batman version enhances something everyone would benefit, this would be worth mentioning.
- If the bugfixes  probably don't have any effect on your gluon network, (not much use mentioning this then)
- If it is unpredictable, if the changes affect our networks,

we should write something like this in the release notes: "backports of batman-adv bugfixes that enhance ..."

File: docs/releases/v2020.2.1.rst [hunk 1]
```diff
@@ -0,0 +1,47 @@
+Gluon 2020.2.1
+===============
+
+Added hardware support
+----------------------
+
+- add support for factory images for TP-Link CPE210 3.20 (`#2080 <https://github.com/freifunk-gluon/gluon/issues/2080>`_)
```
Comment (NeoRaider) [inline docs/releases/v2020.2.1.rst:7]: - Start with an uppercase letter
- This also added the aliases needed for updates
- Let's not mix tenses in the release notes. I think we usually write our release notes in past tense ("Added support ...")

File: docs/releases/v2020.2.1.rst [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+Bugfixes
+--------
+
+- Fixes handling of enabled mesh_on_lan in site configuration (`#2090 <https://github.com/freifunk-gluon/gluon/issues/2090>`_)
```
Comment (NeoRaider) [inline docs/releases/v2020.2.1.rst:12]:
```suggestion
- Fixed handling of *mesh_on_lan* enabled in site configuration (`#2090 <https://github.com/freifunk-gluon/gluon/issues/2090>`_)
```

File: docs/releases/v2020.2.1.rst [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+
+- Fixes build issues with lantiq-xrx200 target by removing unsupported DSL modem packages (`#2087 <https://github.com/freifunk-gluon/gluon/pull/2087>`_)
+
+Other changes
+-------------
+
+- Linux kernel has been updated to 4.14.193
+- backports of batman-adv bugfixes
```
Comment (NeoRaider) [inline docs/releases/v2020.2.1.rst:20]:
```suggestion
- Backports of batman-adv bugfixes
```

**Review by NeoRaider — CHANGES_REQUESTED** [review]

File: docs/releases/v2020.2.1.rst [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+----------------------
+
+- add support for factory images for TP-Link CPE210 3.20 (`#2080 <https://github.com/freifunk-gluon/gluon/issues/2080>`_)
+
+Bugfixes
+--------
+
+- Fixes handling of enabled mesh_on_lan in site configuration (`#2090 <https://github.com/freifunk-gluon/gluon/issues/2090>`_)
+
+- Fixes build issues with lantiq-xrx200 target by removing unsupported DSL modem packages (`#2087 <https://github.com/freifunk-gluon/gluon/pull/2087>`_)
```
Comment (NeoRaider) [inline docs/releases/v2020.2.1.rst:14]: Is this needed in v2020.1.x as well?

Comment (NeoRaider) [reply docs/releases/v2020.2.1.rst:14]: @blocktrron

File: docs/releases/v2020.2.1.rst (hunk 1 above)
Comment (rotanid) [reply docs/releases/v2020.2.1.rst]: done

**Review by NeoRaider — APPROVED** [review]

File: docs/releases/v2020.2.1.rst (hunk 4 above)
Comment (blocktrron) [reply docs/releases/v2020.2.1.rst]: Nope, as v2020.1 does not ship OpenSSL when enabling a Gluon feature.  We should be fine there.

**Review by blocktrron — APPROVED** [review]

**Pull request merged**